### PR TITLE
Updated examples using Template SAML

### PR DESCRIPTION
**Pull request by ganghoonkim-okta**
Updated examples using Template SAML to use the custom SAML app instead.

Resolves: [OKTA-97383](https://oktainc.atlassian.net/browse/OKTA-97383)

@mystiberry-okta @bng-okta please review.

### CODE REVIEW

File: _source/docs/api/resources/apps.md [hunk 1]
```diff
@@ -150,7 +150,6 @@ The catalog is currently not exposed via an API.  While additional apps may be a
 | template_swa        | [Add Plugin SWA Application](#add-plugin-swa-application)                     |
 | template_swa3field  | [Add Plugin SWA (3 Field) Application](#add-plugin-swa-3-field-application)   |
 | tempalte_sps        | [Add SWA Application (No Plugin)](#add-swa-application-no-plugin)             |
-| template_saml_2_0   | [Add SAML 2.0 Application](#add-saml-20-application)                          |
```
Comment (bng-okta) [inline _source/docs/api/resources/apps.md]: We should place this link with the `Custom SAML Application` link instead. Not sure what the right anchor is

Comment (mystiberry-okta) [inline _source/docs/api/resources/apps.md]: Aren't there two, now, the one you deleted and one for #add-custom-saml-application?

Comment (bng-okta) [inline _source/docs/api/resources/apps.md]: yeah there were two before but I don't see the new one (`#add-custom-saml-application`) in this table.

File: _source/docs/api/resources/apps.md [hunk 2]
```diff
@@ -150,8 +150,9 @@ The catalog is currently not exposed via an API.  While additional apps may be a
...
 | template_wsfed      | [Add WS-Federation Application](#add-ws-federation-application)               |
+| Custom SAML 2.0     | [Add Custom SAML 2.0 Application](#add-custom-saml-application)               |
+| Custom SWA          | [Add Custom SWA Application](#add-custom-swa-application)                     |
```
Comment (ganghoonkim-okta) [inline _source/docs/api/resources/apps.md]: @bng-okta @mystiberry-okta added both custom SAML and SWA to the table.

Comment (mystiberry-okta) [inline _source/docs/api/resources/apps.md]: Looks great! Ready to merge?

**Comment by bng-okta** [comment]
squash and :shipit:

**Pull request merged**